src/Tickets/Commerce/Gateways/PayPal/Assets.php

```diff
@@ -94,7 +94,7 @@ public function register() {
 				'tribe-tickets-commerce-base-gateway-checkout-js',
 				'tribe-tickets-commerce-base-gateway-checkout-toggler',
 			],
-			null,
+			'wp_enqueue_scripts',
 			[
 				'groups'       => [
 					'tec-tickets-commerce-gateway-paypal',
```

Comment:
Maybe we want to add the `'tribe-tickets-commerce-checkout'` group here and that could make it.